### PR TITLE
{tools}[intel-2018b] PyFMI 2.4.0 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by hajgato** [comment]
needs PR #6741

File: easybuild/easyconfigs/f/FMILibrary/FMILibrary-2.0.3-intel-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+builddependencies = [
+    ('CMake', '3.11.4'),
+]
+
+configopts = '-DFMILIB_INSTALL_PREFIX=%(installdir)s '
+
+separate_build_dir = True
+
+sanity_check_paths = {
+    'files': ['lib/libfmilib%s' % x for x in ['.a', '_shared.%s'% SHLIB_EXT]],
```
Comment (boegel) [inline easybuild/easyconfigs/f/FMILibrary/FMILibrary-2.0.3-intel-2018b.eb]: @hajgato Style issue being caught by the test: missing space before last `%`

**Comment by boegel** [comment]
@hajgato Is it worth adding `Assimulo` both as stand-alone installation and as extension included in the `PyFMI` bundle?

If you keep `Assimulo` stand-alone, why not use that as a proper dependency for `PyFMI` rather than including it as extension?

**Comment by boegel** [comment]
(close/re-open to re-trigger Travis, I think the dep variants check will fail now on `lxml`...)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2611.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/22ae677c54f73c6c0dddf77fdc43ac81 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/78f95a64d9e16a3bbbf0c26ea583818e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2053.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/bb8deadd5eee7c7ee77a253750607aad for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**